.github/workflows/image-minimizer.js

```diff
@@ -4,7 +4,12 @@
 module.exports = async ({github, context}) => {
     const IGNORE_KEY = '<!-- IGNORE IMAGE MINIFY -->';
     const IGNORE_ALT_NAME_END = 'ignoreImageMinify';
+    // Targeted maximum height
     const IMG_MAX_HEIGHT_PX = 600;
+    // maximum width of GitHub issues/comments
+    const IMG_MAX_WIDTH_PX = 800;
+    // all images that have a lower aspect ration (-> have a smaller width) than this will be minimized
+    const MIN_ASPECT_RATION = IMG_MAX_WIDTH_PX / IMG_MAX_HEIGHT_PX
```

Comment:
It's "ratio," not "ration." The usages of the variable should also be changed.
```suggestion
    // all images that have a lower aspect ratio (-> have a smaller width) than this will be minimized
    const MIN_ASPECT_RATIO = IMG_MAX_WIDTH_PX / IMG_MAX_HEIGHT_PX
```